probot-app-report-benchmarks/index.js

```diff
@@ -188,3 +255,35 @@ module.exports = app => {
     );
   }
 };
+
+function createTableString(topTitles, sideTitles, data) {
+  let outString =
+    "|[Help](https://docs.google.com/document/d/18TpXOEkjlfSVKfjdwJJZSe59Ju4U66sn4oH6egq1O84/edit?usp=sharing)|";
```

Comment:
Where should this link to? Right now its just the rfc for this in google docs, but perhaps there is a better location.